字符串和正则表达式/序列.py

```diff
@@ -0,0 +1,65 @@
+# topical: 序列
+# author: Liu Xiaoxia
+# time: 2020.07.05
+
+# 列表、元组和字符串的共同点：
+# -都可以通过索引得到每一个元素
+# -默认索引值总是从0开始
+# -可以通过分片的方法得到一个范围内的元素的集合
+# -有很多共同的操作符(重复操作符、拼接操作符、成员关系操作符)
+
+# list()把一个可迭代对象转换为列表
+a = list()
+print(a)
+b = "shu jia kai shi la!"
+b = list(b)
+print(b)
+c = (1, 1, 2, 3, 5, 8, 13, 21, 34)
+c = list(c)
+print(c)
+
+# tuple()把一个可迭代对象转换为元组
+# 和list()类似
+
+# str(obj)把obj对象转换为字符串
+
+# len(sub)返回sub的长度
+
+# max()返回序列或者参数集合中的最大值
+# 空序列会报错
+# 保证元素是同一种类型，否则报错
+tuple1 = (1, 2, 3, 4, 5, 9, 0)
+print(max(tuple1))
+print(max('s', 'h', 'u', ' ', 'j', 'i', 'a'))
+numbers = [2, 52, 61, 15, 8, -45]
+print(max(numbers))
+chars = '154236890'
+print(max(chars))
```

Comment:
Note with result is better,such as print('a') **// a**
```suggestion
tuple1 = (1, 2, 3, 4, 5, 9, 0)
print(max(tuple1))
print(max('s', 'h', 'u', ' ', 'j', 'i', 'a'))
numbers = [2, 52, 61, 15, 8, -45]
print(max(numbers))
chars = '154236890'
print(max(chars))
```